### PR TITLE
improve docs a bit

### PR DESCRIPTION
**Pull request by Kiarokh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ Please see CONTRIBUTING for details.
 
 # Credits
 * [Johan Groth](https://github.com/jgroth) — Development
+* [Adrian Schmidt](https://github.com/adrianschmidt) — Development
```
Comment (adrianschmidt) [inline README.md:45]: ❤️

**Review by adrianschmidt — APPROVED** [review]

**Comment by adrianschmidt** [comment]
Hmm… there's a failing test, but it fails on `master` to… 🤔

@jgroth Ping!

**Comment by jgroth** [comment]
> Hmm… there's a failing test, but it fails on `master` to… 
> 
> @jgroth Ping!

Yeah I noticed. I updated some dependencies last week after a discussion I saw on Slack and it seems some where breaking. That's what you get when you merge to `master` and assume it should just work I guess :sweat_smile:

**Comment by jgroth** [comment]
I reverted the latest commit on `master`, will look into why it was failing at a later time :slightly_smiling_face:

**Review by jgroth — APPROVED** [review]

**Pull request merged**